Tests/BackendIntegrationTests/BaseStoreKitIntegrationTests.swift

```diff
@@ -164,6 +171,29 @@ extension BaseStoreKitIntegrationTests {
         return data
     }
 
+    @discardableResult
+    func purchaseShortestDuration(
+        allowOfflineEntitlements: Bool = false,
+        file: FileString = #file,
+        line: UInt = #line
+    ) async throws -> PurchaseResultData {
+        let logger = TestLogHandler()
+        let product = try await StoreKit.Product.products(for: ["shortest_duration"]).first!
```

Comment:
Nit: use the constant here

```suggestion
        let product = try await StoreKit.Product.products(for: [Self.weeklyWith30DayTrial]).first!
```